pages/index.tsx

```diff
@@ -138,38 +140,48 @@ const getStateAndStringFreq = (array: string[]): FilterElem[] => {
 };
 
 const getTagsStateWithFreq = (devfiles: Devfile[]): FilterElem[] => {
-  const tagValues: string[] = devfiles?.map((devfile) => devfile?.tags).flat() as string[];
+  const tagValues = devfiles?.map((devfile) => devfile?.tags).flat() as string[];
 
-  const tagsStateWithFreq: FilterElem[] = getStateAndStringFreq(tagValues);
+  const tagsStateWithFreq = getStateAndStringFreq(tagValues);
 
   return tagsStateWithFreq;
 };
 
 const getTypesStateWithFreq = (devfiles: Devfile[]): FilterElem[] => {
-  const typeValues: string[] = devfiles?.map((devfile) => devfile.type);
+  const typeValues = devfiles?.map((devfile) => devfile.type);
 
-  const tagsStateWithFreq: FilterElem[] = getStateAndStringFreq(typeValues);
+  const tagsStateWithFreq = getStateAndStringFreq(typeValues);
 
   return tagsStateWithFreq;
 };
 
+const getSourceReposStateWithFreq = (devfiles: Devfile[]): FilterElem[] => {
```

Comment:
That is because these values are used the the filter forum on the left hand side. Ill change the name of the function to hopefully make more sense. I never did really like that function name.

Reply:
fixed